Foundation/NSError.swift

```diff
@@ -64,7 +91,16 @@ public class NSError : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
     
     public func encodeWithCoder(aCoder: NSCoder) {
-        
+        if aCoder.allowsKeyedCoding {
+            aCoder.encodeObject(_domain.bridge(), forKey: "NSDomain")
```

Comment:
Docs/Issues.md says that bridge() is temporary so changing API permanently does not seem the right idea